moderation/config.go

```diff
@@ -218,6 +222,8 @@ func configFromModel(model *models.ModerationConfig) *Config {
 		WarnCmdRoles:           model.WarnCmdRoles,
 		WarnIncludeChannelLogs: model.WarnIncludeChannelLogs.Bool,
 		WarnSendToModlog:       model.WarnSendToModlog.Bool,
+		DelwarnSendToModlog:     model.DelwarnSendToModlog.Bool,
+		DelwarnIncludeWarnReason: model.DelwarnIncludeWarnReason.Bool,
```

Comment:
Did you rerun sqlboiler after changing the schema per Soggy's suggestion? Now that these columns are `NOT NULL`, you should not need the `.Bool` anymore; sqlboiler should just generate normal booleans.